corehq/apps/reports/standard/sms.py

```diff
@@ -941,13 +948,21 @@ def rows(self):
             elif messaging_subevent.content_type == MessagingEvent.CONTENT_EMAIL:
                 timestamp = ServerTime(messaging_subevent.date).user_time(self.timezone).done()
                 status = get_status_display(messaging_subevent)
+                content = '-'
+                recipient_address = '-'
+                try:
+                    email = Email.objects.get(messaging_subevent=messaging_subevent.pk)
+                    content = email.body
+                    recipient_address = email.recipient_address
+                except Email.DoesNotExist:
+                    pass
                 result.append([
                     self._fmt_timestamp(timestamp),
                     self._fmt_contact_link(messaging_subevent.recipient_id, doc_info),
-                    self._fmt('-'),
-                    self._fmt('-'),
-                    self._fmt_direction('-'),
-                    self._fmt('-'),
+                    self._fmt(content),
```

Comment:
How is HTML content in the email handled? Does datatables take care of it?

Reply:
I think it just renders it (thats how the links work)